net/mptcp/mptcp_sched.c

```diff
@@ -94,18 +94,22 @@ static bool mptcp_is_temp_unavailable(struct sock *sk,
 
 	return false;
 }
+EXPORT_SYMBOL_GPL(mptcp_is_temp_unavailable);
 
 /* Is the sub-socket sk available to send the skb? */
 bool mptcp_is_available(struct sock *sk, const struct sk_buff *skb,
 			bool zero_wnd_test)
 {
-	return !mptcp_is_def_unavailable(sk) &&
-	       !mptcp_is_temp_unavailable(sk, skb, zero_wnd_test);
+	if (sk)
+		return !mptcp_is_def_unavailable(sk) &&
+		       !mptcp_is_temp_unavailable(sk, skb, zero_wnd_test);
+	else
+		return false;
```

Comment:
Looking through the code in ```mptcp_blest.c``` it does not seem that this change is really needed, right?

Reply:
Good catch! Thank you for reviewing the code. This change is a remainder of the LAMPS scheduler (see https://github.com/lampsmptcp/lamps). In our case it is completely non functional and can be reverted.